packages/databricks-vscode/resources/whats-new/2.5.md

```diff
@@ -0,0 +1,67 @@
+<img src="./1.2/databricks-logo.png" alt="databricks-logo" width="100"/>
+
+# Databricks Extension v2.5.0
+
+## What's new?
+
+-   [Validation and partial update actions for pipelines](#validation)
+-   [Pipeline events and diagnostics](#events)
+-   [Pipeline datasets and their schema](#datasets)
```

Comment:
```suggestion
-   [Pipeline datasets and their schemas](#datasets)
```
Do we need "and their schemas" here? Isn't supporting datasets assume schemas are included?

Reply:
Will remove it